.github/workflows/docker-build.yml

```diff
@@ -281,21 +281,34 @@ jobs:
         id: slug
       
       - name: Login to Docker Hub
-        if: github.event_name != 'pull_request'
+        #if: github.event_name != 'pull_request'
```

Comment:
We still need the guard. The push action will succeed without logging in if push is disabled.